examples/basic/MODULE.bazel

```diff
@@ -1,2 +1,6 @@
-bazel_dep(name = "pybind11_bazel", version = "2.12.0")
 bazel_dep(name = "rules_python", version = "0.33.2")
+bazel_dep(name = "pybind11_bazel")
+local_path_override(
+    module_name = "pybind11_bazel",
+    path = "..",
```

Comment:
Isn't `..` relative to the `examples/basic/` directory... the `examples/` directory? :)

Reply:
Good catch, fixed